.prettierrc

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
you goal is to use this file for print and frontend?

People like me run the npm commands from the docker container, for that you would need to mount this file in the print and the frontend container.

But People like me can fix that themselves later

Reply:
I'd see this as a requirement, too.
Idea: Could we move the `.prettierrc` file into `/common`, and then symlink from frontend, from print and from the root folder? The common folder is already mounted as volume in both the print container and the frontend container.

Reply:
I would say mounting the file with docker-compose directly is enough and simple.

Reply:
Implemented in c9f130a